### PR TITLE
Link to current file/folder in `forked-to` links

### LINKED ISSUE #2810: "View file on fork" link in file view
E.g. file view page, like:
https://github.com/sindresorhus/refined-github/blob/master/source/options.html
would have a "View file on fork" link (button) which would redirect you to the file on your fork, i.e. for me it would be (assuming I would have a fork of this repo):
https://github.com/jack1142/refined-github/blob/master/source/options.html
If user doesn't have the fork yet, either the button should not be available or it should also create the fork for the user.

### PR DESCRIPTION
**Pull request by yakov116**
1. LINKED ISSUES:
	Closes #2810

2. TEST URLS:
   Test on any fork

3. SCREENSHOT:
   None

### CODE REVIEW

File: source/features/forked-to.tsx [hunk 1]
```diff
@@ -29,6 +30,21 @@ const updateCache = cache.function(async (): Promise<string[] | undefined> => {
...
+			user,
+			repository,
+			branch: 'HEAD'
+		});
+
+		return url.pathname;
+	}
+
+	return '/' + baseRepo;
+}
```
Comment (fregante) [inline source/features/forked-to.tsx:46]:
```suggestion
	if (pageDetect.isRepoRoot() || !(pageDetect.isSingleFile() || pageDetect.isRepoTree())) {
		return '/' + baseRepo;
	}
	
	const [user, repository] = baseRepo.split('/');
	const url = new GitHubURL(location.href).assign({
		user,
		repository,
		branch: 'HEAD'
	});

	return url.pathname;
}
```

**Comment by fregante** [comment]
> If yes what should it be called `reverse-forked-to` 😄

It's completely unrelated to `forked-to`

**Comment by yakov116** [comment]
> It's completely unrelated to `forked-to`

I was making a joke.

**Comment by fregante** [comment]
```
Revert "I was making a joke."
```

**Pull request merged**